charts/camunda-platform-8.8/templates/common/ingress-grpc.yaml

```diff
@@ -34,21 +34,19 @@ spec:
                 port:
                   number: {{ .Values.orchestration.service.grpcPort }}
   {{- if .Values.orchestration.ingress.grpc.tls.enabled }}
-  {{- if and 
-    (not .Values.orchestration.ingress.grpc.tls.secretName) 
-    (contains "openshift-"  (default "" .Values.orchestration.ingress.grpc.className)) 
-  }}
-    # The tls block is not applied because secretName is empty and className contains "openshift-".
-    # This is necessary to use the certificate managed by the OpenShift Ingress operator,
-    # which will automatically convert this Ingress into a Route (https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/security_and_compliance/configuring-certificates#replacing-default-ingress_replacing-default-ingress).
-  {{- else }}
   tls:
+    {{- if and (not .Values.orchestration.ingress.grpc.tls.secretName) (contains "openshift-" (default "" .Values.orchestration.ingress.grpc.className)) }}
+    # For OpenShift: Use empty TLS block to allow OpenShift Ingress Operator to manage certificates automatically.
```

Comment:
We need to convert this to a template comment using `{{/* ... */}}` so it's not rendered in the manifest.